.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+Please start by choosing an appropriate base branch for your pull request.
```

Comment:
I would make the next two points here bullets.

- ``production`` branch: Quick fixes to problems or documentation enhancement
- ``master`` branch: Introduction of new features